.pre-commit-config.yaml

```diff
@@ -132,3 +132,7 @@ repos:
         exclude: *fixtures
         args: ["--max-summary-lines=2", "--linewrap-full-docstring"]
         files: "pylint"
+  - repo: https://github.com/regebro/pyroma
+    rev: "4.1"
+    hooks:
+      - id: pyroma
```

Comment:
I don't really follow the project's README? Why would you want to run this every commit?

Reply:
Right it's always running because of this https://github.com/regebro/pyroma/blob/master/.pre-commit-hooks.yaml#L8. Let me fix that.